docs/useMutation.md

```diff
@@ -100,6 +100,15 @@ The `mutationState` object has the following properties:
 - `data`: the response data for the mutation
 - `error`: any errors returned by the mutation
 
+## Optimistic Updates
```

Comment:
add this section also in this file:
https://github.com/relay-tools/relay-hooks/blob/master/useMutation.md

Reply:
Quick question, why maintain 2 copies of the same files? 

The links in the README could link to the files in the `docs` folder to make it less redundant probably?

Reply:
@shreyas44 great suggestion;)
can you do it in this PR? ;)

Reply:
I think it would be better to open a separate PR as it's for a different purpose?

Reply:
you're right but I didn't want to bother you doing extra work

Reply:
Ahh, it's no problem! Just created a new PR for this #157 🙂